public/static/docs/command-reference/diff.md

```diff
@@ -1,47 +1,45 @@
 # diff
 
-Show changes between versions of the <abbr>DVC project</abbr>. It can be
-narrowed down to specific target files and directories under DVC control.
+Show changes between <abbr>DVC repository</abbr>
+[revisions](https://git-scm.com/docs/revisions). It can be narrowed down to
+specific target files and directories under DVC control.
 
-> This command requires that the project is a [Git](https://git-scm.com/)
-> repository.
+> This command requires that the <abbr>project</abbr> is a
+> [Git](https://git-scm.com/) repository.
 
 ## Synopsis
 
 ```usage
 usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
 
 positional arguments:
-  a_ref                 Git reference from which diff calculates
-  b_ref                 Git reference until which diff calculates, if
-                        omitted diff shows the difference between
-                        current HEAD and a_ref
+  a_ref     Git reference from which the diff begins
+  b_ref     Git reference until which the diff ends. If omitted,
+            `HEAD` (latest commit) is used.
 ```
 
 ## Description
 
-Given two Git commit references (commit hash, branch or tag name, etc) `a_ref`
-and `b_ref`, this command shows a a summary of basic statistics: how many files
-were deleted/changed, and the file size differences.
+Given two commit SHA hashes, branch or tag names, etc.
+([Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References))
```

Comment:
so, is it a reference of revision?

Reply:
In this case the param is `a_ref` so we use the term reference. But references point to revisions, so it's both.

Reply:
so, we are about to change this in dvc itself. so, can start changing here as well.

Reply:
You mean to "commit". In a recent comment you mentioned maybe we don't want to change "revision" to "commit" yet? This PR is already pretty complicated, if you don't mind I'll wait until this changes in the core repo.

Reply:
so, it still better to not use that link to references already in this repo, no matter how params are called?

Reply:
Sure, replaced them with https://github.com/iterative/dvc.org/pull/933#discussion_r373753614 in 4d7c9c0b